### PR TITLE
add mssql table definition

### PR DESCRIPTION
**Pull request by krismorte**
Table script and Compatibility Test

### CODE REVIEW

File: src/test/resources/mssql_tables.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+create table scheduled_tasks (
+  task_name varchar(250) not null,
+  task_instance varchar(250) not null,
+  task_data  nvarchar(max),
+  execution_time datetimeoffset  not null,
+  picked varchar(10),
```
Comment (kagkarlsson) [inline src/test/resources/mssql_tables.sql]: Isn't there a boolean or bit type in mssql?

**Comment by kagkarlsson** [comment]
Looks better without the whitespace diffs, thanks :)

Did the schema work with the `CompatibilityTest`?

File: src/test/resources/mssql_tables.sql (hunk 1 above)
Comment (krismorte) [reply src/test/resources/mssql_tables.sql]: usually is used bit type

**Comment by krismorte** [comment]
When I run the `CompatibilityTest` it's show me some error in the schema. So I change the table schema and run again without errors.

File: src/test/resources/mssql_tables.sql (hunk 1 above)
Comment (kagkarlsson) [reply src/test/resources/mssql_tables.sql]: If it works with `bit` that is better

Comment (krismorte) [reply src/test/resources/mssql_tables.sql]: I had not looked at this particular column before. I ran the CompatibilityTest again and the result was 0, so I changed the column to bit and the result as the same. I will submit another commit.

**Comment by kagkarlsson** [comment]
Ok, I will merge this and comment out the tests and the dependency (similar to what I did for Oracle). Will try to setup integration-tests against all supported datasources when I have time.

**Pull request merged**